docs/getting-started/clients/desktop/mac/index.md

```diff
@@ -52,7 +52,7 @@ security add-generic-password -a "<apple_id>" -w "<app_specific_password>" -s "A
 
 ### Provisioning Profile
 
-1. Ask DevOps (@DevOps in slack) to have your `Apple Development` signing certificate added to the
+1. Ask DevOps (@BRE in slack) to have your `Apple Development` signing certificate added to the
```

Comment:
```suggestion
1. Ask BRE (@bre in Slack) to have your `Apple Development` signing certificate added to the
```

Reply:
whoops! Agreed we should update as suggested.